apps/omg_watcher_info/test/omg_watcher_info/api/block_test.exs

```diff
@@ -0,0 +1,56 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherInfo.API.BlockTest do
+  use ExUnitFixtures
+  use ExUnit.Case, async: false
+  use OMG.WatcherInfo.Fixtures
+
+  import OMG.WatcherInfo.Factory
+
+  alias OMG.Utils.Paginator
+  alias OMG.WatcherInfo.API
+  alias OMG.WatcherInfo.DB
+
+  describe "get_blocks/1" do
+    @tag fixtures: [:phoenix_ecto_sandbox]
+    test "returns a paginator with a list of blocks" do
```

Comment:
Wamt tp add more data checking similar to this test: apps/omg_watcher_rpc/test/omg_watcher_rpc/web/controllers/block_test.exs

Reply:
This is pretty much all that `OMG.WatcherInfo.API.Block.get_blocks/1` does. Those other tests can go as tests at the paginator and controller level.